backend/src/Service/Subscription/SubscriptionService.php

```diff
@@ -362,4 +376,21 @@ public function canSubscriptionExtra(string $status, bool $type): bool|null
       
       return SubscriptionConstant::CAN_SUBSCRIPTION_EXTRA_FALSE;
     }
+
+    public function isPackageReadyForSubscription(int $packageId): SubscriptionErrorResponse
```

Comment:
I guess we do not have to autoMapping here, as long as the function aim is just to return a string which indicate if the package is exist or not.